internal/pkg/auth/storage.go

```diff
@@ -69,6 +71,15 @@ func SetAuthField(key authFieldKey, value string) error {
 }
 
 func setAuthFieldInKeyring(key authFieldKey, value string) error {
+	activeProfile, err := config.GetProfile()
+	if err != nil {
+		return fmt.Errorf("get profile: %w", err)
+	}
```

Comment:
Would it make sense to move these to `SetAuthFeld` and pass the profile as an argument? Same for the getters

Reply:
We can discuss this once you're back as it would have no influence in the behaviour